### PR TITLE
Boldwin: update maintainer email

### PR DESCRIPTION
**Pull request by Wls-demo**

### CODE REVIEW

File: static/bidder-info/boldwin.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 endpoint: "http://ssp.videowalldirect.com/pserver"
 maintainer:
-  email: "wls_demo_box@smartyads.com"
+  email: "info@bold-win.com"
```
Comment (gargcreation1992) [inline static/bidder-info/boldwin.yaml:3]: Sent email for verification.

Comment (gargcreation1992) [reply static/bidder-info/boldwin.yaml:3]: We haven't received a reply yet. Please resend. @Wls-demo

Comment (Wls-demo) [reply static/bidder-info/boldwin.yaml:3]: @gargcreation1992 we have replied, please check

**Comment by Wls-demo** [comment]
@gargcreation1992 are there any other problems?

File: static/bidder-info/boldwin.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/boldwin.yaml:3]: received. thanks!

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**